MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -220,8 +220,13 @@ public GraphicsDevice(GraphicsAdapter adapter, GraphicsProfile graphicsProfile,
             _graphicsProfile = graphicsProfile;
             Setup();
             GraphicsCapabilities = new GraphicsCapabilities();
-            GraphicsCapabilities.Initialize(this);
-            Initialize();
+            GraphicsCapabilities.Initialize(this);
+
+#if WINDOWS
+            CorrectBackBufferSize();
+#endif
```

Comment:
Can we do this in a better way that doesn't involve an `#if WINDOWS` here?  We are trying to reduce the `#if`s and not increase them.